packages/react/src/create-element.js

```diff
@@ -20,9 +20,21 @@ const MDXCreateElement = forwardRef((props, ref) => {
 
   const components = useMDXComponents(propComponents)
   const type = mdxType
+
+  const getMemberExpressionElement = () => {
+    if (!type.includes('.')) {
+      return
+    }
+
+    const [objName, propName] = type.split('.')
```

Comment:
Will this *not* handle `<Foo.Bar.Baz />` or am I misunderstanding this?

Reply:
It won't. This is mostly a temporary monkey patch that will be better addressed in the v2 branch.

Reply:
if we wanted this to be more fully supported we could copy/pasta lodash's `get` code, or something similar: https://lodash.com/docs/4.17.15#get